doc/download.html

```diff
@@ -84,21 +84,24 @@ <h2 align="center">Downloading TestNG</h2>
 <h3>Eclipse plug-in</h3>
 <p>You can use either the Eclipse Market Place or the update site:</p>
 <ul>
-	<li>Select<i> Help / Install new software.</i> 
-	</li>
-	<li><i>Search for new features to install.</i> 
-	</li>
-	<li><i>New remote site.</i> 
-	</li>
-	<li>For Eclipse 3.4 and above, enter <a href="http://beust.com/eclipse">http://beust.com/eclipse</a>.
-	<li>For Eclipse 3.3 and below, enter <a href="http://beust.com/eclipse1">http://beust.com/eclipse1</a>.
-	</li>
-	<li>Make sure the check box next to URL is checked and click <i>Next</i>. 
-	</li>
-	<li>Eclipse will then guide you through the process. </li>
+  <li>Select<i> Help / Install new software.</i></li>
+  <li><i>Search for new features to install.</i></li>
+  <li><i>New remote site.</i></li>
+  <li>Eclipse 4.2 and above - Enter <a href="http://beust.com/eclipse">http://beust.com/eclipse</a>.</li>
+  <li>Eclipse 3.x - Not supported any more, please update your Eclipse to 4.2 or above.
+    <ul>
```

Comment:
Just remove the whole "If you unfortunately" section, no point in giving recommendations on things we won't support.